src/content/learn/add-react-to-an-existing-project.md

```diff
@@ -62,60 +62,60 @@ Then add these lines of code at the top of your main JavaScript file (it might b
 <html>
   <head><title>My app</title></head>
   <body>
-    <!-- Your existing page content (in this example, it gets replaced) -->
+    <!-- Váš stávající obsah stránky (v tomto příkladu bude nahrazen) -->
   </body>
 </html>
 ```
 
 ```js index.js active
 import { createRoot } from 'react-dom/client';
 
-// Clear the existing HTML content
+// Nahradí existující obsah HTML
 document.body.innerHTML = '<div id="app"></div>';
 
-// Render your React component instead
+// Do něj nyní vykreslíme React komponentu
 const root = createRoot(document.getElementById('app'));
-root.render(<h1>Hello, world</h1>);
+root.render(<h1>Ahoj světe</h1>);
 ```
 
 </Sandpack>
 
-If the entire content of your page was replaced by a "Hello, world!", everything worked! Keep reading.
+Pokud byl celý obsah stránky nahrazen nápisem "Ahoj světe", vše funguje správně! Čtěte dál.
 
 <Note>
 
-Integrating a modular JavaScript environment into an existing project for the first time can feel intimidating, but it's worth it! If you get stuck, try our [community resources](/community) or the [Vite Chat](https://chat.vitejs.dev/).
+První integrace modulárního prostředí JavaScriptu do existujícího projektu může působit zastrašujícím dojmem, ale výsledky stojí za to! Pokud narazíte na problémy, využijte naše [komunitní zdroje ](/community) nebo se obrate na [Vite Chat](https://chat.vitejs.dev/).
 
 </Note>
 
-### Step 2: Render React components anywhere on the page {/*step-2-render-react-components-anywhere-on-the-page*/}
+### Krok 2: Vykreslení React komponenty na libovolném místě na stránce {/*step-2-render-react-components-anywhere-on-the-page*/}
 
-In the previous step, you put this code at the top of your main file:
+V předchozím kroku jste tento kód umístili na začátek hlavního souboru:
 
 ```js
 import { createRoot } from 'react-dom/client';
 
-// Clear the existing HTML content
+// Nahradí existující obsah HTML
 document.body.innerHTML = '<div id="app"></div>';
 
-// Render your React component instead
+// Do něj nyní vykreslíme React komponentu
 const root = createRoot(document.getElementById('app'));
-root.render(<h1>Hello, world</h1>);
+root.render(<h1>Ahoj světe</h1>);
 ```
 
-Of course, you don't actually want to clear the existing HTML content!
+Samozřejmě, v reálném scénáři nebudete chtít existující obsah HTML vymazat!
```

Comment:
Maybe different form of the sentence. The "reálném scrénáři" sounds a bit off tome. Maybe "reálném případě" could also fit.
```suggestion
Samozřejmě, v reálu nebudete chtít existující obsah HTML vymazat!
```